plugins/telemetry/math_conversions.cpp

```diff
@@ -21,7 +22,7 @@ Telemetry::EulerAngle to_euler_angle_from_quaternion(Telemetry::Quaternion quate
     return euler_angle;
 }
 
-Telemetry::Quaternion to_quaternion_from_euler_angle(Telemetry::EulerAngle euler_angle)
+Quaternion to_quaternion_from_euler_angle(EulerAngle euler_angle)
```

Comment:
I miss the namespacing of these structs now and I fear they will then be included in other plugins as well, and not just inside telemetry.

Reply:
One question could also be: should they go in `core`? Are they not datastructs that we'd like to share publicly at the library level? Or is it really a concept of the telemetry module?

Reply:
Yer, we could have them shared as well, that's true.